spec/models/web_hook_spec.rb

```diff
@@ -64,6 +64,17 @@
 
         WebHook.enqueue_hooks(:post, post_id: 1)
       end
+
+      it 'registers related event on DiscourseEvent' do
```

Comment:
Do you feel it's a bit awkward to build these specs? Given a model spec, it's not easy to control all function calls' parameters. Even expects a job may be conflicts with others. Like `PostCreator` enqueues a lot of jobs.